anushajain19/README.md

```diff
@@ -0,0 +1,27 @@
+ #100DaysOfCode
+
+**Anusha Jain**
+
+*Second year Electrical Engineering, MNIT Jaipur*
+
+---
+
+## Goals
+
+**Competitive Programming**
+
+- () solve most of competitive programming problems.
```

Comment:
Hey Anusha!
Please use the markdown syntax for checkbox.
Use [this](https://www.markdownguide.org/extended-syntax/#task-lists) for reference.

Reply:
Boss is it alright now

Reply:
Cool. Do you plan to do something for [this](https://github.com/Codeshows/100DaysOfCode/pull/25#discussion_r362191909) too or is it ready to merge?

Reply:
You did not make any changes to this branch :)